suites/squid/nvmeof/tier-3_4-group_2-gw_128-sub_1024-NS.yaml

```diff
@@ -0,0 +1,461 @@
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node11
+          - node12
+          - node13
+          - node14
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client for NVMe tests
+      polarion-id: CEPH-83573758
+
+  - test:
+      abort-on-fail: false
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: shell
+              args:
+                - ceph osd pool create nvmeof_pool
+          - config:
+              command: shell
+              args:
+                - rbd pool init nvmeof_pool
+          - config:
+              command: apply
+              service: nvmeof
+              args:
+                placement:
+                  nodes:
+                  - node3
+                  - node4
+              pos_args:
+                - nvmeof_pool
+                - group1
+          - config:
+              command: shell
+              args:
+                - ceph osd pool create rbd1
+          - config:
+              command: shell
+              args:
+                - rbd pool init rbd1
+      desc: deploy NVMeoF service for GW group 1
+      destroy-cluster: false
+      do-not-skip-tc: true
+      module: test_cephadm.py
+      name: deploy NVMeoF service for GW group 1
+      polarion-id: CEPH-83595696
+
+  - test:
+      abort-on-fail: false
+      config:
+        node: node3
+        rbd_pool: rbd1
+        do_not_create_image: true
+        rep-pool-only: true
+        steps:
+          - config:
+              service: subsystem
+              command: add
+              args:
+                subsystems: 128
+                max-namespaces: 1024
+          - config:
+              service: listener
+              command: add
+              args:
+                subsystems: 128
+                port: 4420
+                group: group1
+                nodes:
+                  - node3
+                  - node4
+          - config:
+              service: host
+              command: add
+              args:
+                subsystems: 128
+                group: group1
+          - config:
+              service: namespace
+              command: add
+              args:
+                subsystems: 128
+                namespaces: 1024
+                group: group1
```

Comment:
Do we have any reason for having `group` property in every config.

Reply:
yes, we append this to subsystem nqn for 100s of subsystem and 4 different groups that are required for listener, host and namespace add

Reply:
okay, can we have it as a config property like at L118

Reply:
I thought of making this as common to all configs but since we have 100s and 1000s of iterations making each time to clear args and again add plus changes to --no-group-append and without it made complicated keeping it outside each config